tests/StackExchange.Redis.Tests/Expiry.cs

```diff
@@ -51,30 +51,30 @@ public async Task TestBasicExpiryTimeSpan(bool disablePTimes)
     [Theory]
     [InlineData(true)]
     [InlineData(false)]
-    public void TestExpiryOptions(bool disablePTimes)
+    public async Task TestExpiryOptions(bool disablePTimes)
     {
         using var conn = Create(disabledCommands: GetMap(disablePTimes), require: RedisFeatures.v7_0_0_rc1);
 
         var key = Me();
-        var cb = conn.GetDatabase();
-        cb.KeyDelete(key, CommandFlags.FireAndForget);
-        cb.StringSet(key, "value", flags: CommandFlags.FireAndForget);
+        var db = conn.GetDatabase();
+        db.KeyDelete(key, CommandFlags.FireAndForget);
+        db.StringSet(key, "value", flags: CommandFlags.FireAndForget);
```

Comment:
sync/async shouldn't matter here, but let's damn really damn sure the key is fresh on this test since it matters heavily
```suggestion
        db.KeyDelete(key);
        db.StringSet(key, "value");
```